src/adapters/scene.js

```diff
@@ -166,7 +166,7 @@ Scene.prototype.initMapBox = function() {
 
   listen('map_mark_poi', (poi, options) => {
     this.ensureMarkerIsVisible(poi, options);
-    if (!options.isFromCategory) {
+    if (!options.resource || !options.resource.category) {
```

Comment:
I'm not a fan of this `resource` name, it sounds really too vague and looks like it describes a single piece of data, not filtering criteria.
What about something like `poiFilters`, `poiParams`, `criteria`?

Reply:
I agree, I was not sure about how to name this value last name and went with `resource`, trying to keep the actual behavior first. I renamed to `poiFilters`